idd/Energy+.idd.in

```diff
@@ -45619,7 +45615,131 @@ Coil:Heating:DX:VariableSpeed,
+  A2, \field Indoor Air Cooling Coil Inlet Node Name
+       \required-field
+       \type alpha
+  A3, \field Indoor Air Heating Coil Inlet Node Name
+       \required-field
+       \type alpha
+  A4, \field Indoor Air Outlet Node Name
+       \required-field
+       \type alpha
+  A5, \field Condenser Water Inlet Node
+       \required-field
+       \type alpha
+  A6, \field Condenser Water Outlet Node
+       \required-field
+       \type alpha
+  A7, \field Water Node to Monitor the Supply Hot Water Flow
+       \required-field
+       \type alpha
+  A8, \field Outdoor Air Inlet Node
+       \required-field
+       \type alpha
+  A9, \field Outdoor Air Outlet Node
+       \required-field
+       \type alpha
+  A10, \ field Coil Name of Space Cooling Mode
+       \required-field
+       \type object-list
+       \object-list CoolingCoilsDXVariableSpeed
+       \note Must match the name used in the corresponding Coil:Cooling:DX:VariableSpeed object.
+  A11,  \field Coil Name of Space Heating Mode
+       \type object-list
+       \object-list HeatingCoilsDXVariableSpeed
+       \note Must match the name used in the corresponding Coil:Heating:DX:VariableSpeed object.
+  A12,  \field Coil Name of Dedicated Water Heating Mode
+       \type object-list
+       \object-list HeatPumpWaterHeaterDXCoils
+       \note Must match the name used in the corresponding Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed object.
+  A13, \field Coil Name of Combined Space Cooling and Water Heating Mode with Full Condensing
+       \type object-list
+       \object-list HeatPumpWaterHeaterDXCoils
+       \note Must match the name used in the corresponding Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed object.
+  A14, \field Cooling Coil Name of Combined Space Cooling and Water Heating Mode with Desuperheating
+       \type object-list
+       \object-list CoolingCoilsDXVariableSpeed
+       \note Must match the name used in the corresponding Coil:Cooling:DX:VariableSpeed object.
+  A15, \field Water Heating Coil Name of Combined Space Cooling and Water Heating Mode with Desuperheating
+       \type object-list
+       \object-list HeatPumpWaterHeaterDXCoils
+       \note Must match the name used in the corresponding Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed object.	   
+  A16, \field Heating Coil Name of Combined Space Heating and Water Heating Mode with Desuperheating
+       \type object-list
+       \object-list HeatingCoilsDXVariableSpeed
+	   \note Must match the name used in the corresponding Coil:Heating:DX:VariableSpeed object.   
+  A17, \field Water Heating Coil Name of Combined Space Heating and Water Heating Mode with Desuperheating
+       \type object-list
+       \object-list HeatPumpWaterHeaterDXCoils
+       \note Must match the name used in the corresponding Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed object.	   	   
+  N1 , \field Indoor Temperature Limit for SCWH Mode
```

Comment:
These are difficult.  If I remember correctly, I pushed back on really long names in the original NFP.  I will wait for a set of revised field names throughout and then review them.  Words like "of" and "is", "to", "above which" should be gone from most field names, if not all.